service/service.py

```diff
@@ -168,19 +174,9 @@ def action_recommend_product(productId):
     """ Route to recommend a list of suppliers given a product"""
     app.logger.info('Recommend product')
     suppliers = Supplier.action_make_recommendation(productId)
-    if suppliers is not None:
+    if len(suppliers) > 0:
         return make_response(suppliers.to_json(), status.HTTP_200_OK)
-    return make_response("NOT FOUND", status.HTTP_404_NOT_FOUND)
-
-
-@app.route('/suppliers', methods=['GET'])
-def query_a_supplier():
```

Comment:
Query function missing?

Reply:
I put query operation into list operation. 
Like the professor's code..

Reply:
OK cool!